.eslintrc.js

```diff
@@ -1,7 +1,6 @@
 /** @format */
-const { useE2EEsLintConfig } = require( '@woocommerce/e2e-environment' );
 
-module.exports = useE2EEsLintConfig( {
+module.exports = {
```

Comment:
@Konamiman Your PR description doesn't explain why you removed this. The reason for this function call is related to running E2E tests.

Reply:
It seemed to me that this was about litning the js only and it was not really needed, but I'll take a closer look, thanks.

Reply:
Ok, looks like that `require( '@woocommerce/e2e-environment' );` isn't actually needed to run the e2e tests, and having it in place prevents the e2e infrastructure to be removed from the build. So indeed it needs to go out, I'll added that explanation to the PR description.

Reply:
Sorry, when I said running E2E tests I generalized that it was needed for E2E. The linter is the js equivalent of phpcs for writing E2E tests. Instead of removing this can you change the import to 

```
const { useE2EEsLintConfig } = require( './tests/e2e/env/config/use-config' );
```

That will allow you to eliminate the package but keep the existing linting process in place.

Reply:
Thanks for your suggestion, but by applying it I still get this when running `grunt`: `Error: Cannot find module '@woocommerce/e2e-environment'`

Reply:
This log is based against the latest on this branch

```
$ git diff
diff --git a/.eslintrc.js b/.eslintrc.js
index 7c90a4dbb..c64f82197 100644
--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -1,6 +1,7 @@
 /** @format */
+const { useE2EEsLintConfig } = require( './tests/e2e/env/config/use-config' );
 
-module.exports = {
+module.exports = useE2EEsLintConfig( {
        root: true,
        env: {
                browser: true,
@@ -28,4 +29,4 @@ module.exports = {
                        jsx: true
                }
        },
-};
+} );
$ grunt
Running "eslint:src" (eslint) task
WARNING: No configurations found in configuration directory:/Users/ronrennick/Sites/solaris/woocommerce/config
WARNING: To disable this warning set SUPPRESS_NO_CONFIG_WARNING in the environment.

/Users/ronrennick/Sites/solaris/woocommerce/assets/js/admin/system-status.js
  76:5  warning  Unexpected console statement  no-console

/Users/ronrennick/Sites/solaris/woocommerce/assets/js/frontend/checkout.js
  493:9  warning  Unexpected console statement  no-console
  495:9  warning  Unexpected console statement  no-console
  496:9  warning  Unexpected console statement  no-console
  499:9  warning  Unexpected console statement  no-console
  670:7  warning  Unexpected console statement  no-console

✖ 6 problems (0 errors, 6 warnings)


Running "uglify:js_assets" (uglify) task
>> 70 files created 1.56 MB → 700.46 kB

Running "sass:compile" (sass) task

Running "rtlcss:woocommerce" (rtlcss) task
File assets/css/activation-rtl.css created.
File assets/css/admin-rtl.css created.
File assets/css/auth-rtl.css created.
File assets/css/dashboard-rtl.css created.
File assets/css/helper-rtl.css created.
File assets/css/marketplace-suggestions-rtl.css created.
File assets/css/menu-rtl.css created.
File assets/css/network-order-widget-rtl.css created.
File assets/css/prettyPhoto-rtl.css created.
File assets/css/privacy-rtl.css created.
File assets/css/reports-print-rtl.css created.
File assets/css/twenty-nineteen-rtl.css created.
File assets/css/twenty-seventeen-rtl.css created.
File assets/css/twenty-twenty-one-admin-rtl.css created.
File assets/css/twenty-twenty-one-rtl.css created.
File assets/css/twenty-twenty-rtl.css created.
File assets/css/wc-setup-rtl.css created.
File assets/css/woocommerce-layout-rtl.css created.
File assets/css/woocommerce-smallscreen-rtl.css created.
File assets/css/woocommerce-rtl.css created.

Running "postcss:dist" (postcss) task
>> 41 processed stylesheets created.

Running "cssmin:minify" (cssmin) task
>> 43 files created. 1.51 MB → 1.24 MB

Running "concat:admin" (concat) task

Done.
```

Reply:
I must have something misconfigured then. Anyway, I have pushed the change you suggested; could someone else from @woocommerce/proton test this please?